src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php

```diff
@@ -84,9 +84,9 @@ public function testCsrfAliases()
         $processor = new Processor();
         $configuration = new MainConfiguration(array(), array());
         $processedConfig = $processor->processConfiguration($configuration, array($config));
-        $this->assertTrue(isset($processedConfig['firewalls']['stub']['logout']['csrf_token_generator']));
+        $this->assertArrayHasKey('csrf_token_generator', $processedConfig['firewalls']['stub']['logout']);
```

Comment:
this change is not better as let;
`$processedConfig['firewalls']['stub']['logout']['csrf_token_generator'] = null`
then in the old situation the test would fail, in the new one it passes.

Reply:
@SpacePossum PR?